pimd/pim_igmp.c

```diff
@@ -753,6 +753,38 @@ static void igmp_group_free(struct igmp_group *group)
 	XFREE(MTYPE_PIM_IGMP_GROUP, group);
 }
 
+static void igmp_group_count_incr(struct igmp_sock *igmp)
+{
+	struct pim_interface *pim_ifp = igmp->interface->info;
+
+	if (!pim_ifp)
+		return;
+
+	++pim_ifp->pim->igmp_group_count;
+	if (pim_ifp->pim->igmp_group_count
+	    == pim_ifp->pim->igmp_watermark_limit) {
+		zlog_warn(
+			"SCALE ALERT: igmp group count reached watermark limit: %u(vrf: %s)",
```

Comment:
can we print vrf related stuff with the format like `vrf %s(%u)`? It's related to https://github.com/FRRouting/frr/pull/5988.

Reply:
SCALE ALERT does not sound good :D

Reply:
IGMP-WM-WARN: sounds good? I remember seeing similar in cisco device.
Let me change it to VRF_LOGNAME(vrf) instead of calling vrf->name.